src/org/omegat/gui/editor/mark/AltTranslationsMarker.java

```diff
@@ -0,0 +1,71 @@
+/**************************************************************************
+ OmegaT - Computer Assisted Translation (CAT) tool
+ with fuzzy matching, translation memory, keyword search,
+ glossaries, and translation leveraging into updated projects.
+
+ Copyright (C) 2024 Lev Abashkin
+               Home page: https://www.omegat.org/
+               Support center: https://omegat.org/support
+
+ This file is part of OmegaT.
+
+ OmegaT is free software: you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+ OmegaT is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ **************************************************************************/
+
+package org.omegat.gui.editor.mark;
+
+import org.omegat.core.Core;
+import org.omegat.core.data.SourceTextEntry;
+import org.omegat.gui.editor.EditorSettings;
+import org.omegat.util.gui.Styles;
+
+import javax.swing.text.Highlighter;
+import java.util.Collections;
+import java.util.List;
+
+public class AltTranslationsMarker extends AbstractMarker {
+
+    protected static final Highlighter.HighlightPainter PAINTER = new TransparentHighlightPainter(
```

Comment:
```suggestion
    private final Highlighter.HighlightPainter painter = new TransparentHighlightPainter(
```
It should not be static, because it can break the marker to work as plugin.